### PR TITLE
Quickstart template for cache override actions via Rulese Engine for Azure CDN end points.

### PR DESCRIPTION
**Pull request by gball14**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use a parameter for resource locations with the defaultValue set to resourceGroup().location
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values)
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md

- [x] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

File: 201-cdn-with-ruleseengine-cacheoverride/README.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+# Simple deployment of cache override via rules engine for a CDN end point
+
+<IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/201-cdn-with-ruleseengine-cacheoverride/PublicLastTestDate.svg" />&nbsp;
+<IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/201-cdn-with-ruleseengine-cacheoverride/PublicDeployment.svg" />&nbsp;
+
+<IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/201-cdn-with-ruleseengine-cacheoverride/FairfaxLastTestDate.svg" />&nbsp;
+<IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/201-cdn-with-ruleseengine-cacheoverride/FairfaxDeployment.svg" />&nbsp;
+
+<IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/201-cdn-with-ruleseengine-cacheoverride/BestPracticeResult.svg" />&nbsp;
+<IMG SRC="https://azurequickstartsservice.blob.core.windows.net/badges/201-cdn-with-ruleseengine-cacheoverride/CredScanResult.svg" />&nbsp;
```
Comment (ganga369) [inline 201-cdn-with-ruleseengine-cacheoverride/README.md]: Not sure how to create the images. All examples have these images but don't see any documentation on how to upload the images.

**Comment by ganga369** [comment]
@ShirleyJ27 is added to the review. #Closed

**Comment by ganga369** [comment]
@hytao is added to the review. #Closed

**Comment by hytao** [comment]
Could you create an example for http to https redirection and Device based UrlRewrite? I think we need more examples.

**Comment by gball14** [comment]
> Could you create an example for http to https redirection and Device based UrlRewrite? I think we need more examples.

I want to submit one example and understand the review process. I will submit more examples as a seperate change.

**Comment by ganga369** [comment]
@bmoore-msft is added to the review. #Closed

**Comment by gball14** [comment]
> @bmoore-msft is added to the review. #Closed

Hi Brian, I added you explicitly as I see your name in the previous pull requests for this repo. I was waiting github to automatically assign the reviewers but that didn't happen. Let me know if there is some process I should follow.

**Comment by gball14** [comment]
#sign-off

**Pull request merged**